ZetaPushSwift/Core/Macro/ZetaPushMacroListener.swift

```diff
@@ -19,7 +19,7 @@ open class ZetaPushMacroListener {
   // MARK: Lifecycle
   public init(_ clientHelper: ClientHelper, deploymentId: String) {
     self.clientHelper = clientHelper
-    self.zetaPushMacroService = ZetaPushMacroService(clientHelper, deploymentId: deploymentId)
+    self.zetaPushMacroService = ZetaPushMacroService(clientHelper, deploymentId: deploymentId, timeOut: clientHelper.timeOut)
```

Comment:
you inject client helper so it is not necessary to create a param to init timeout